Sources/Sentry/SentryOptions.m

```diff
@@ -56,6 +56,7 @@ - (void)setMeasurement:(SentryMeasurementValue *)measurement
 {
     NSMutableArray<NSString *> *defaultIntegrations =
```

Comment:
`H`: add a note that the order of integrations is important and must not be changed